File: FilmArchive.Domain/Models/Photo.cs
namespace FilmArchive.Domain.Models
{
    public class Photo
    {
        public int Id { get; set; }
        public string Url { get; set; }
        public Camera Camera { get; set; }
        public Film Film { get; set; }
        public int ExposureIndex { get; set; }
        public string Location { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Country { get; set; }
    }
}

File: FilmArchive.WebUI/ClientApp/src/app/models/Photo/photo.ts
import { Camera } from "../Camera/camera";
import { Film } from "../Film/film";

export class Photo {
  id: number;
  url: string;
  camera: Camera;
  film: Film;
  exposureIndex: number;
  location: string;
  city: string;
  state: string;
  country: string;
}
